nucliadb_protos/utils.proto

```diff
@@ -47,16 +47,17 @@ message RelationNode {
     string subtype = 6;
 }
 
-message JoinGraphCnx {
+message JoinGraphEdge {
```

Comment:
Is this pb stored in maindb? if so, renaming could bring us problems with already created data?

Reply:
AFAIK it's not stored, in fact, I think it haven't been used anywhere yet. But correct me if I'm wrong @bloodbare